client/client.go

```diff
@@ -18,19 +18,22 @@ import (
 	"github.com/charmbracelet/keygen"
 	"github.com/golang-jwt/jwt/v4"
 	"github.com/mitchellh/go-homedir"
+	gap "github.com/muesli/go-app-paths"
 	"golang.org/x/crypto/ssh"
 )
 
 var nameValidator = regexp.MustCompile("^[a-zA-Z0-9]{1,50}$")
 
 // Config contains the Charm client configuration.
 type Config struct {
-	Host     string `env:"CHARM_HOST" envDefault:"cloud.charm.sh"`
-	SSHPort  int    `env:"CHARM_SSH_PORT" envDefault:"35353"`
-	HTTPPort int    `env:"CHARM_HTTP_PORT" envDefault:"35354"`
-	Debug    bool   `env:"CHARM_DEBUG" envDefault:"false"`
-	Logfile  string `env:"CHARM_LOGFILE" envDefault:""`
-	KeyType  string `env:"CHARM_KEY_TYPE" envDefault:"ed25519"`
+	Host        string `env:"CHARM_HOST" envDefault:"cloud.charm.sh"`
+	SSHPort     int    `env:"CHARM_SSH_PORT" envDefault:"35353"`
+	HTTPPort    int    `env:"CHARM_HTTP_PORT" envDefault:"35354"`
+	Debug       bool   `env:"CHARM_DEBUG" envDefault:"false"`
+	Logfile     string `env:"CHARM_LOGFILE" envDefault:""`
+	KeyType     string `env:"CHARM_KEY_TYPE" envDefault:"ed25519"`
+	DataDir     string `env:"CHARM_DATA_DIR" envDefault:""`
+	IdentityKey string `env:"CHARM_IDENTITY_KEY" envDefault:""`
```

Comment:
if you prefer, you can drop the `CHARM_` prefix and pass it as an option when parsing

e.g.:

```go
	var c Config
	if err := env.Parse(&c, env.Options{
		Prefix: "CHARM_",
	}); err != nil {
		panic(err)
	}
```